src/sentry/apidocs/parameters.py

```diff
@@ -94,6 +94,16 @@ def member_id(description: str) -> OpenApiParameter:
         )
 
 
+class EnvironmentParams:
+    VISIBILITY = OpenApiParameter(
+        name="visibility",
+        location="query",
+        required=False,
+        type=str,
+        description="""The visibility of the environments to filter by. The options are: `all`, `hidden`, `visible`. Defaults to `visible`.""",
```

Comment:
How are hidden & visible determined? Is it through which projects a user has access to or is it manually set per environment?

Reply:
its manually set per environment

Reply:
Ah gotcha